### PR TITLE
DOC: Add new and retired Council members

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: src/pages/council.js
Comment (dzenanz) [inline src/pages/council.js:51]: Use lowercase `ć`:
```suggestion
          <li>Dženan Zukić - Kitware</li>
```

**Review by dzenanz — APPROVED** [review]

File: src/pages/council.js
Comment (jhlegarreta) [inline src/pages/council.js:49]: Ah, yes, thanks Matt :-). A typo in the name of the hospital: Brigham and Women's Hospital

**Review by jhlegarreta — COMMENTED** [review]
Thanks for doing this Matt.

**Review by blowekamp — APPROVED** [review]

File: src/pages/council.js
Comment (thewtex) [reply src/pages/council.js]: Thanks for the review @jhlegarreta, addressed

**Comment by thewtex** [comment]
> Maybe a post in discourse would be helpful in spreading the news?

Yes, will do.

@jjomier @sbonaretti please take a look.

**Comment by jjomier** [comment]
@thewtex can we postponed until I get the green light on my side (hopefully in the next week)? Thanks.

**Comment by thewtex** [comment]
@jjomier yes

**Comment by sbonaretti** [comment]
Looks good to me! Thanks! 
Let me know if you need more!

**Review by jhlegarreta — APPROVED** [review]

**Review by sbonaretti — APPROVED** [review]

**Comment by jhlegarreta** [comment]
I now realize that we still do not have a confirmation from @jjomier: any news? Shall I revert the commit?

**Comment by thewtex** [comment]
Yes, @jjomier is still waiting on approval. This should not have been merged. We can make changes as-needed later.

**Comment by jhlegarreta** [comment]
Sorry @thewtex , my bad. I push forced the `master` branch to the immediately previous commit; looks like a new PR will need to be opened with these changes, and hopefully be merged after Julien reports back.

**Comment by thewtex** [comment]
Julien has received the go-ahead -- updated PR here: https://github.com/InsightSoftwareConsortium/insightsoftwareconsortium.org/pull/83

**Pull request merged**
